pyEpiabm/pyEpiabm/tests/test_unit/test_sweep/test_queue_sweep.py

```diff
@@ -58,6 +59,67 @@ def test_call(self):
         self.assertEqual(self.person2.time_of_status_change,
                          self.time)
 
+    def test_vaccine_protection_full(self):
+        """Tests that a vaccinated person can be moved to the vaccinated
+        compartment.
```

Comment:
Should they not already be there? Maybe this will make more sense once I've seen the wiki

Reply:
The doc string is probably a bit unclear here. I'm testing that when people are removed from the queue to be vaccinated their infection status is updated accordingly.